xar-assembly.xml

```diff
@@ -18,7 +18,7 @@
     <tag>application</tag>
     <category id="apps">Applications</category>
 
-    <dependency processor="http://exist-db.org" semver-min="${exist.processor.version}" semver-max="6"/>
+    <dependency processor="http://exist-db.org" semver-min="6.1.0" semver-max="6"/>
```

Comment:
This should still be `${exist.processor.version}` no?

Reply:
Good catch! I am actually still unsure if it wouldn't be much nicer and more flexible to have the entire dependency hard-coded here.